### ISSUE TITLE
Notified of repository creation I'm not a part of

### ISSUE THREAD

**Issue by colinhumber** [scrapyomama/LinkedIn-Network-Booster#1]
@scrapyomama Hi, I was notified at my email address that this repository was created. I'm not sure why, since my user isn't linked to this at all. Do you have any idea why I was notified?

**Comment by sameehj** [comment]
@colinhumber Same issue, I was notified that I am invited to collaborate on the Linkedin-Profile-Scrapper project.

**Comment by sameehj** [comment]
@colinhumber Definitely, but it seems harmless. Now that Microsoft acquired Github and Linkedin that's scary :)

**Comment by colinhumber** [comment]
@sameehj I'm just more concerned why I was notified when there's no indication on the repository that either of us have anything to do with it. I posted an issue on the other repo on this account and the author subsequently closed it without any notice, so makes me think something sketchy is going on.

**Comment by ursuleacv** [comment]
I also got a notification
